### PR TITLE
Announcement for Gaprindashvili GA

### PR DESCRIPTION
**Pull request by cybette**
@JPrause @jvlcek @abellotti @gtanzillo @chrispy1 please review 

Included the mention of miqldap_to_sssd

(still working on updating the download links, but please check the blog post first. Thanks!)

### CODE REVIEW

**Comment by JPrause** [comment]
It's more than FINE,..it LGTM! 👍

File: site/_posts/2018-01-30-manageiq-gaprindashvili-ga-announcement.html.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+In oVirt, we have targeted refresh improvements for templates and host events. The OVN (Open Virtual Network) has been added as oVirt's network provider for network switch visualization and control.
+
+And as a shift away from real time metrics for alerts, ManageIQ can now receive push alerts from Prometheus in OpenShift. The Prometheus Alerts Collection Manager is added to Container Providers, and visible through the Monitor -> Alerts menu.
+
+[![Prometheus Alerts](/assets/images/blog/Gaprindashvili_GA_Prometheus_alerts.png)](/assets/images/blog/Gaprindashvili_GA_Prometheus_alerts.png)
+
+## All Good Things...
+
+We are deprecating the MiqLdap client in this release, which provides support for the Authentication Mode: LDAP and LDAPS. An automated conversion script, call `miqldap_to_sssd`, is provided to convert ManageIQ Authentication configurations that use the MiqLdap client to using External authentication with SSSD. See this [blog post](/blog/2017/09/miqldap-to-sssd/) describing how to use the `miqldap_to_sssd` conversion tool.
```
Comment (gtanzillo) [inline site/_posts/2018-01-30-manageiq-gaprindashvili-ga-announcement.html.md]: > conversion script, call `miqldap_to_sssd`

The word "call" should be "called"

Comment (jvlcek) [reply site/_posts/2018-01-30-manageiq-gaprindashvili-ga-announcement.html.md]: Looks good.

Just one nit:

I suggest rewording:

An automated conversion script, call `miqldap_to_sssd`, is provided to convert ManageIQ Authentication configurations that use the MiqLdap client to using External authentication with SSSD.

To be:

An automated conversion script, call `miqldap_to_sssd`, is provided that will  convert ManageIQ Authentication configurations from using the MiqLdap client into Authentication configurations that will use External authentication with SSSD.

**Comment by JPrause** [comment]
Was there anything else needed. I'd like to see if we can merge this today.

**Comment by cybette** [comment]
@JPrause I think I've added everything. tested locally, looks good.

@epwinchell @chessbyte can either of you please merge, thanks!
